CheapShopWeb/DataContext/ProductDbContext.cs

```diff
@@ -11,5 +11,25 @@ public class ProductDbContext : DbContext
     {
         public ProductDbContext() : base(nameOrConnectionString: "connection"){}
         public virtual DbSet<Product> Products { get; set; }
+
```

Comment:
Nemanau kad šis metodas turėtų būti šioje klasėje

Reply:
Ką tuo noriu pasakyti, kad šioje klasėje negali būti jokios logikos (bent jau taip galvociau)

Reply:
Okey, tiesa, tuojau patvarkysiu